store/key_value_store_server.cc

```diff
@@ -35,12 +34,57 @@ bool KvstoreServiceImpl::fileExists(const char *fileName)
 }
 
 void KvstoreServiceImpl::CreateKeyFile(const std::string &key, const std::string &value){
+  if (!fileExists("key-pair.txt")){
+    std::ofstream myfile;
+    myfile.open("key-pair.txt", std::ios_base::app);
+    myfile << key << ":" << value << "\n";
+    myfile.close();
+  } else {
+    std::fstream infile("key-pair.txt");
+    std::string line;
+    std::string delimiter = ":";
```

Comment:
You may want to look at using protobuf's SerializeTo() and ParseFrom() for file storage. This will take care of delimiting, so you do not have to deal with keys or values potentially containing the delimiter.

Reply:
Thanks! Will add that to the list